share/nisar/defaults/static.yaml

```
@@ -0,0 +1,436 @@
+      # Defaults to 'L'.
+      radar_band: L
+      # [OPTIONAL] Data processing center.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/processingCenter' dataset in the output
+      # product.
+      # Defaults to '(NOT SPECIFIED)'.
+      processing_center: (NOT SPECIFIED)
+      # [OPTIONAL] UTC date and time of the start of the validity date range for the
+      # resulting product, as a string in ISO 8601 format with integer seconds
+      # precision.
+      # The output STATIC product is valid for use with NISAR products with observation
+      # datetimes not earlier than this datetime but earlier than the validity start
+      # datetime of any future superseding STATIC product.
+      # Defaults to '1970-01-01T00:00:00'.
+      # WARNING: Due to limitations with the way this YAML file is parsed, datetimes
+      # with sub-microsecond precision will be truncated to integer microseconds
+      # precision unless surrounded in quotes.
+      validity_start_datetime: 1970-01-01T00:00:00
+      # [OPTIONAL] Product counter.
+      # An integer in the range [1, 999].
+      # May be used to distinguish multiple products generated with the same inputs and
+      # software version. Should be reset to 1 whenever the CRID is updated.
+      # Defaults to 1.
+      product_counter: 1
+
+    geometry:
+      # [OPTIONAL] Integer identifier of the corresponding NISAR track.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/trackNumber' dataset in the output product.
+      # Defaults to 0.
+      relative_orbit_number: 0
+      # [OPTIONAL] Integer identifier of the corresponding NISAR frame.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/frameNumber' dataset in the output product.
+      # Defaults to 0.
+      frame_number: 0
+
+    processing:
+      geo_grid:
+        # [OPTIONAL] EPSG code indicating the spatial reference system of the desired
+        # output geographic coordinate grid.
+        # Must be a valid integer EPSG code supported by ISCE3.
+        # Defaults to the same EPSG code as the input DEM raster file.
+        epsg:
+        # [OPTIONAL] X and Y coordinates of the upper-left corner pixel in the output
+        # grid, in the coordinate system and units described by `epsg`.
+        # If `epsg` was specified and differs from the DEM raster EPSG code, these
+        # parameters are required. Otherwise, they default to the coordinates of the
+        # upper-left corner of the input DEM raster.
+        top_left:
+          x:
+          y:
+        # [OPTIONAL] X and Y coordinates of the lower-right corner pixel in the output
+        # grid, in the coordinate system and units described by `epsg`.
+        # Must satisfy `bottom_right.x` > `top_left.x` and `bottom_right.y` <
+        # `top_left.y`.
```

Within the sentence, would you be open to having "left" and "right" correspond to the left and right sides of the inequality?

```suggestion
        # Must satisfy `top_left.x` < `bottom_right.x` and `top_left.y` >
        # `bottom_right.y`.
```